pkg/build/gobuild.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/mutate"
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 	"github.com/google/go-containerregistry/pkg/v1/types"
+	goreleaserCfg "github.com/goreleaser/goreleaser/pkg/config"
```

Comment:
I think I would prefer copying what we actually need from this package, i.e. [spork](https://twitter.com/jonjonsonjr/status/1372415386516262915) it.

Importing this changes the dependency graph quite a bit, and we basically only need a couple structs and a serialization method (I think).

Maybe we start with this under `internal/goreleaser` for now?

Reply:
Thanks for this feedback. I was annoyed by the changes, too. It will be ... sporked. You should probably provide a declension table. Would be fitting if it is an irregular verb.

Reply:
Quick question: I sporked the required pieces into the `ko` repository. Which boilerplate would be applied to the file?

Reply:
Keep theirs, we've done something similar with k8s stuff: https://github.com/google/go-containerregistry/blob/main/internal/retry/wait/kubernetes_apimachinery_wait.go